inloop/common/templatetags/markdown.py

```diff
@@ -8,21 +8,99 @@
 .. _Markdown package extensions
    https://python-markdown.github.io/extensions/
 """
+import subprocess
 
 from django.template import Library
 from django.template.defaultfilters import stringfilter
 from django.utils.safestring import mark_safe
 
-from markdown import Markdown
+from markdown import Extension, Markdown
 from markdown.extensions.codehilite import CodeHiliteExtension
+from markdown.treeprocessors import Treeprocessor
+
+from inloop import settings
```

Comment:
```suggestion
from django.conf import settings
```

@martinmo I just found out, that it is possible to directly suggest changes on GitHub. See [here under 6](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request).